mis_builder_budget/models/mis_budget_by_account_item.py

```diff
@@ -5,7 +5,11 @@
 
 
 class MisBudgetByAccountItem(models.Model):
-    _inherit = ["mis.budget.item.abstract", "prorata.read_group.mixin"]
+    _inherit = [
+        "mis.budget.item.abstract",
+        "prorata.read_group.mixin",
+        "analytic.mixin",
```

Comment:
Adding the analytic mixin goes beyond the scope of a migration. Can we do this in another PR?

Reply:
Sure: https://github.com/OCA/mis-builder/pull/634